api/kubernetes/helm-chart/kube-prometheus-stack.yaml

```diff
@@ -0,0 +1,72 @@
+namespace: prometheus
+name: cosmotech-api-latest
+defaultRules:
+  create: false
+alertmanager:
+  enabled: false
+grafana:
+  enabled: false
+kubeApiServer:
+  enabled: false
+kubelet:
+  enabled: false
+kubeControllerManager:
+  enabled: false
+coreDns:
+  enabled: false
+kubeEtcd:
+  enabled: false
+kubeScheduler:
+  enabled: false
+kubeStateMetrics:
+  enabled: false
+nodeExporter:
+  enabled: false
+prometheus:
+  enabled: true
+  crname: prometheus
+  serviceAccount:
+    create: true
+    name: prometheus-service-account
+  prometheusSpec:
+    logLevel: info
+    replicas: 1
+    podMetadata:
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+      labels:
+        app: prometheus
+    resources:
+      limits:
+        cpu: 1
+        memory: 2Gi
+      requests:
+        cpu: 1
+        memory: 2Gi
+    retention: 12h
+    serviceMonitorSelector:
+      matchLabels:
+        serviceMonitorSelector: prometheus
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: "standard"
```

Comment:
This should be a variable. Usually it is "default" but in kind it is "standard".